src/main/java/io/javalin/websocket/WsContext.kt

```diff
@@ -9,18 +9,18 @@ package io.javalin.websocket
 import io.javalin.core.util.ContextUtil
 import org.eclipse.jetty.websocket.api.*
 import org.eclipse.jetty.websocket.common.WebSocketSession
-import java.net.InetSocketAddress
 import java.nio.ByteBuffer
 
 /**
- * The [WsSession] class is a wrapper for Jetty's [Session].
+ * The [WsContext] class holds Jetty's [Session] and provides (convenient) delegate methods.
  * It adds functionality similar to the API found in [io.javalin.Context].
  * It also adds a [send] method, which calls [RemoteEndpoint.sendString] on [Session.getRemote]
  */
-class WsSession(val id: String, session: Session, private var pathParamMap: Map<String, String>, private val matchedPath: String) : Session {
+class WsContext(val id: String, session: Session, private var pathParamMap: Map<String, String>, private val matchedPath: String) {
```

Comment:
Should we rename `val id: String` into `val sessionid: String`?

Reply:
Yes, good idea.